snuba/utils/streams/batching.py

```diff
@@ -113,25 +113,23 @@ def __init__(
         self.__batch: Optional[Batch[TResult]] = None
         self.__closed = False
 
-    def process(self, message: Optional[Message[TPayload]]) -> None:
+    def poll(self) -> None:
```

Comment:
I am not sure this is the right name for the method now. From `poll` in general I would expect a result, while this is actually running a cycle on the strategy. What about simply `execute` or `run`?

Reply:
👍  this was initially a bit confusing for me too

Reply:
I doubt we will find a better name (@lynnagara any idea ?) I would be ok with the additional doc.

Reply:
Sounds good to add some additional doc and keep the name, especially if there is some precedent for it I wasn't aware of